lib/mix/lib/mix/task.ex

```diff
@@ -360,6 +375,19 @@ defmodule Mix.Task do
     end
   end
 
+  def run_requirements(module) do
+    case Mix.Task.requirements(module) do
+      nil ->
+        nil
+
+      requirements ->
```

Comment:
This is probably clearer as an `if`:

```suggestion
    if requirements = requirements(module) do
```